src/miner_lora.erl

```diff
@@ -174,6 +198,8 @@ reg_domain_data_for_countrycode(CC)->
 
 -spec region() -> {ok, atom()}.
 region()->
+    %% TODO: recalc region if hotspot re-asserts
+    %% region_us915, region_au915
```

Comment:
What does `region_us915, region_au915` mean here?

Reply:
Removed in 7809680